src/app/game-state/item-repo.service.ts

```diff
@@ -52,11 +53,12 @@ export class ItemRepoService {
       type: 'furniture',
       slot: 'bed',
       value: 10000,
+      previousId: 'mat',
       description: "A fine bed with a cover. Curtains keep the mosquitoes off you during the night. Increases daily stamina recovery by 2 and restores a bit of health.",
       useConsumes: false,
       use: () => {
         this.characterService.characterState.status.stamina.value += 2;
-        this.characterService.characterState.status.health.value += 0.02;
+        this.characterService.characterState.status.health.value += 0.2;
```

Comment:
This _seems_ like a typo since the previous bed (blanket) restores 0.1. Let me know @immortalityidle

Reply:
Yeah, that was just me being sloppy.